src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root')
-);
-
```

Comment:
- [ ] Inside `index.js` you should call a component called `App.js` according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Once%20the%20Calculator%20is%20ready%2C%20import%20it%20and%20use%20it%20in%20your%20main%20component%20(App.js).). To use an `App.js` is a good practice inside a webpack project. 🐱 
- Remember, `App.js` should be a class-based component.
- `App.js` should have `Calculator.js` inside of it.